tests/src/test/scala/org/apache/openwhisk/core/admin/WskAdminTests.scala

```diff
@@ -252,7 +252,8 @@ class WskAdminTests extends TestHelpers with WskActorSystem with Matchers with B
       // check correctly set
       val lines = wskadmin.cli(Seq("limits", "get", subject)).stdout.linesIterator.toSeq
       lines should have size 1
-      lines(0) shouldBe "allowedKinds = [u'nodejs:10', u'blackbox']"
+      lines(0) should (be("allowedKinds = [u'nodejs:10', u'blackbox']") or be(
+        "allowedKinds = ['nodejs:10', 'blackbox']"))
```

Comment:
* If use python3 as default: `ln -s  /usr/bin/python3  /usr/bin/python`
   `lines(0)` is `['nodejs:10', 'blackbox']`
* If use python2 as default: `ln -s  /usr/bin/python2  /usr/bin/python`
   `lines(0)` is `[u'nodejs:10', u'blackbox']`
So here meets 2 cases.